nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss feature mediaicons 21

# Description - created Media Icons component

## Type of change


# Testing Instructions
Fetch branch and serve it to see if media icons component shows up in DevCard and then click on each icon to make sure it opens new tab and takes you to the link and opens email upon clicking

# Checklist:
